src/Symfony/Component/Form/FormView.php

```diff
@@ -20,6 +20,8 @@ class FormView implements \ArrayAccess, \IteratorAggregate, \Countable
 {
     /**
      * The variables assigned to this view.
+     *
+     * @var mixed
```

Comment:
this is wrong. It is always an array, not a mixed value (a full psalm type would be `array<string, mixed>` but we don't use that notation in Symfony, so this should be `@var array`)

Reply:
@stof agree, changed it